buildSrc/subprojects/integration-testing/src/main/kotlin/org/gradle/gradlebuild/test/integrationtests/shared-configuration.kt

```diff
@@ -106,11 +107,20 @@ fun Project.integrationTestUsesSampleDir(vararg sampleDirs: String) {
 }
 
 
+/**
+ * kotlin-dsl-plugins integration test depends on jars published to local repository
+ */
 fun Project.integrationTestUsesKotlinDslPlugins() {
     tasks.withType<IntegrationTest>().configureEach {
-        inputs.dir(project(":kotlinDslPlugins").buildDir.resolve("repository"))
-            .withPropertyName("localRepository")
+        inputs.files(callable {
```

Comment:
Do we need to do a similar input normalization for "libsRepository.required = true"? Or is that already done elsewhere?